lib/ansible/module_utils/pycompat24.py

```diff
@@ -44,10 +44,21 @@ def get_exception():
             e = get_exception()
 
     """
+    deprecate(
```

Comment:
@sivel what's the correct deprecation syntax? pylint seems to be unhappy with specifying the version arg...

Reply:
The version is the version it will be removed in, not the version it was deprecated in.

Reply:
Oh, I didn't realize from that linter output..